prisma/schema.prisma

```diff
@@ -0,0 +1,66 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Statement {
+  id            String @id @unique @default(cuid())
+  actorId       String
+  actorType     ObjectType?
+  verbId        String
+  objectId      String
+  objectType    ObjectType?
+  timestamp     DateTime?
+  stored        DateTime @default(now())
+  version       String @default("1.0.0")
+  statement     Json
+  attachments   Json?
+}
+
+enum ObjectType {
+  Agent
+  Group
+  Activity
+  StatementRef
+  SubStatement
+}
+model AgentProfile {
+  id            String @id
+  objectType    ObjectType?
+  account       AgentAccount?
+  mbox          String?
```

Comment:
we have been using "camelCase" in most parts of the API, any reason for not using `mBox`?

Reply:
Same for lines 39 and 40.

Reply:
Spec for Inverse Functional Identifiers does not use camelCase. Up to us if we want to in internal code though
https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Data.md#2423-inverse-functional-identifier